.pylintrc

```diff
@@ -0,0 +1,6 @@
+[MESSAGE CONTROL]
+disable = abstract-method,anomalous-backslash-in-string,invalid-all-object,lost-exception,missing-docstring,too-few-public-methods
+
+good-names = db,id
+function-rgx = [a-z_][a-z0-9_]{2,60}$
+method-rgx = [a-z_][a-z0-9_]{2,60}$
```

Comment:
sometimes I think if we should have a centralized pylint/pyflake/etc. to run on all projects that want to use it like we do with https://github.com/elifesciences/proofreader-php
but this is a social more than technical problem, getting everyone to agree on the configuration

Reply:
I agree. I just matched what https://github.com/elifesciences/profiles had for the most part.

Centralized base configs would be great.